processor/processor.go

```diff
@@ -667,6 +667,7 @@ func (proc *HandleT) getFailedEventJobs(response transformer.ResponseT, metadata
 			CreatedAt:    time.Now(),
 			ExpireAt:     time.Now(),
 			CustomVal:    metadata.DestinationType,
+			UserID:       failedEvent.Metadata.UserID, // will be nil if it went throgh user transformation
```

Comment:
through - typo

Instead of having null userID, can we somehow pass one of the userIDs. If it is a null userID, it might not be picked by multi-node migrator ever.